src/app/AppConfig.ts

```diff
@@ -59,19 +64,26 @@ export type AppConfig = {
 	// Nothing yet...
 }
 
+export type AppConfigKey = keyof AppConfig
```

Comment:
doesn't seem to make much difference for me, also TSC doesn't complain without it

Reply:
Added usage of it in other places